### DISCUSSION TITLE
Love it

### DISCUSSION THREAD

**Discussion by diwasatreya** [community/community#13509] [Other Feature Feedback, Questions, & Ideas]
I love this new feature. It would help me to find codes for my projects

**Comment by jfullstackdev** [comment]
there is a new update on the For You feed, you can now filter it, and for those who don't want it can simply disable it by unchecking all filters

![image](https://user-images.githubusercontent.com/47092464/182012117-0b19be9d-e849-462e-909e-d20d28915c1d.png)
